changelog.d/pr-11014

```diff
@@ -0,0 +1,7 @@
+---
+synopsis: 'add SPDX license data version 3.26'
```

Comment:
```suggestion
synopsis: add SPDX license data version 3.26
```

these quotes shouldn't be necessary?..

Reply:
I cribbed from an existing changelog entry, which uses markdown-frontmatter. I should convert it to a normal one since I don't need Markdown.